### DISCUSSION TITLE
DiscordOverwriteBuilder() obsolete why?

### DISCUSSION THREAD

**Discussion by eyagovbusiness** [DSharpPlus/DSharpPlus#1471] [Support] [answered]
Hello, since DiscordOverwriteBuilder() constructor is marked as obsolete for the version 4.4, I am wondering how I could construct a new DiscordOverwriteBuilder from DiscordOverwriteBuilder. FromAsync(), could we get FromAsync() static? even in 4.3 would be great to have DiscordOverwriteBuilder. FromAsync(DiscordOverwrite other) instead of needing an instance of DiscordOverwriteBuilder to call the method. 

Thank you everyone who made possible this project :)

**Comment by akiraveliara** [comment, marked as answer]
use one of the specialized constructors, it's just the one that's obsolete
